packages/api/app/Middleware/Permission.js

```diff
@@ -2,9 +2,11 @@ const UnauthorizedException = use('App/Exceptions/UnauthorizedException');
 const Permission = use('App/Models/Permission');
 
 class PermissionMiddleware {
-	async handle({ auth, params }, next, properties) {
+	async handle({ auth, request }, next, properties) {
+		const { params } = request;
+		const reqParams = request.all();
 		const user = await auth.getUser();
-		const isAuthorized = await Permission.checkPermission(user, properties, params);
+		const isAuthorized = await Permission.checkPermission(user, properties, params, reqParams);
```

Comment:
Pq nao mergear o params com o reqParams? Não soa bem adicionar um novo parâmetro para cada tipo de origem dos filtros...
```suggestion
		const isAuthorized = await Permission.checkPermission(user, properties, { ...params, ...reqParams });
```
Isso faria demais mudanças relacionadas a esse parâmetro desnecessárias 👍

Reply:
Eu pensei nisso no inicio mateus, mas eu tive um problema pq em alguns casos pode existir o mesmo parâmetro em ambos, ai quando vc faz o merge tem problema.